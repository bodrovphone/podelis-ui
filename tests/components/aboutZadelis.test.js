import React from "react";
import "@testing-library/jest-dom/extend-expect";
// Using render and screen from test-utils.js instead of
// @testing-library/react

import { render, screen } from "../test-utils";
import AboutZadelis from "../../components/aboutZadelis";

describe("AboutZadelis", () => {
  it("should render AboutZadelis", () => {
    render(<AboutZadelis />);

    const heading = screen.getByText(
      /Zadils - Простой и удобный шаринг-сервис\./i
    );

    // we can only use toBeInTheDocument because it was imported
    // in the jest.setup.js and configured in jest.config.js
    expect(heading).toMatchSnapshot();
    // .toBeInTheDocument();
  });
});
