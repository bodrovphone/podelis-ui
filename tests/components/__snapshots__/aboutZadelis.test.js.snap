// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutZadelis should render AboutZadelis 1`] = `
<div
  class="styles__Content-qezad7-1 bLPDXH"
>
  Zadils - Простой и удобный шаринг-сервис.
  <br />
  Арендуй поблизости.
  <br />
  Пользуйся тем что нужно, не покупая.
  <br />
  Сдавай в аренду вещи и зарабатывай.
  <br />
</div>
`;
